### ISSUE TITLE
Minor Review Responses

### ISSUE THREAD

**Issue by katyhuff** [cyclus/fundamentals-paper#115]
From Reviewer 1, this includes:

- [ ] The paper gives an overall good overview of the motivations for and the development of CYCLUS. **Solution:** no changes necessary
- [ ] IA: would be good to mention that such simulators can be used both for questions addressing operational facets of nuclear energy systems, e.g. interim UNF storage optimisation,as well as 'strategic' questions allowing to address R&D-options assessment. **Solution:** mention this fact
- [ ] IB.iii: The simulation codes with discrete materials tracking are: COSI, FAMILY21, GENIUS 1 and 2, DANESS, NFCSim and ORION. DESAE doesn't !! Not clear about VISION. DYMOND didn't neither. **Solution:** remove desae and add orion.

From Reviewer 2:

- [ ] The manuscript has the following title: "Fundamental Concepts in the Cyclus Fuel Cycle Simulator Framework and Modeling Ecosystem". It is not clear to me what "and Modeling Ecosystem" refers to. **Solution:** Remove modeling ecosystem from the title, but add a mention of this notion in the abstract
- [ ] page 3 (Background): "With few exceptions, these metrics are derived from mass balances calculated by a fuel cycle simulator. [4]." : It would be nice to illustrate what are the few exceptions. Please read "C. Poinssot, S. Bourg, N. Ouvrier, N. Combernoux, C. Rostaing, M. Vargas-Gonzalez et J. Bruno, Assessment of the environmental footprint of nuclear energy systems. Comparison between closed and open fuel cycles., Energy, n°1In Press, pp. 1-13, 2014. **Solution:** cite C. Poinssot.
- [ ] page 18: "SWUs" please give a complete list of acronyms: here Separate Work Units **Solution:** expand SWU (this is why we have the glossary package, folks!!!)
- [ ] page 20: bad reference for:  "Section ?? " **Solution:** fix ref.
- [ ] page 20: What is a BatchReactor? **Solution:** Point out in the response that this term is defined immediately after its first mention on this page. I don't think this needs to be solved. However, we can point out that it's deprecated in 1.3 and we can add a parenthetical where it is mentioned on the third line.
- [ ] page 23: typing error: "…and it cannot…" **Solution:** Fix typo.

**Comment by rwcarlsen** [comment]
on item IB.iii, maybe we just need to be more clear about what we mean by discrete material tracking - because I am fairly confident that what we mean by it in Cyclus is significantly different than what the reviewer is thinking.  And as far a my understanding goes for the other codes, they don't do discrete material tracking.  Some of the codes may have various pieces - i.e. knowing how long some material was in a reactor, remembering a "fuel type", or even remembering how long material has been at some location.  But they don't track and preserve material object identicy completely like we do in Cyclus.

**Comment by gidden** [comment]
@rwcarlsen are you suggesting we claim that Cyclus' "discrete material tracking" is significantly different than other codes that have that capability? While I don't doubt that there are different "flavors" and we might have the most feature-complete "flavor", this is likely not a battle we want to fight. I think the suggested fix is reasonable, but would of course be open to more discussion.

**Comment by gidden** [comment]
Regarding Poinssot et. al. (note they use 'indicator' rather than metric):
```
They are the green-house-gases emissions (GHG, gCO2eq/kWhe) [13] and [21], the atmospheric pollution (SOx and NOx, mg/kWhe) [21], the water pollution (mg/kWhe) [21], the land-use (m2/GWhe) [13], the water consumption and withdrawal (L/MWhe) [13] and [22], and the production of technological waste (g/MWhe) [23].

...

They are the radioactive gaseous and liquid releases (Bq/kWhe) and the solid radioactive waste production (g/MWhe or m3/MWhe) [21]. The footprint of the required repository is also considered.

... acidification, eutrophication and POCP (photochemical ozone creation potential) phenomena, ...,  eco-toxicity and human toxicity was used...
```

It seems like the reviewer would include only the middle group as actual fuel-related mass balances. If that is true and we cite this paper, I don't think we can say `With few exceptions`. Many of the others could be thought of as functions of mass flow (via power density) though...

**Comment by gonuke** [comment]
On discrete material tracking... If we are going to declare that some codes have it and others don't, then we should probably define the criteria that we are using.  In so doing, we could find a way to acknowledge various implementations of all or part of what we there define.

**Comment by gonuke** [comment]
On Poinssot...  I generally also include "facility operation history" with "mass balances" as the primary data that is necessary to derive the majority of indicators/metrics.  Would this help?

**Comment by gonuke** [comment]
Regarding BatchReactor: If it doesn't produce substantive changes, I think it's fair game to update this section to be consistent with 1.3.

**Comment by katyhuff** [comment]
re: poinssot @gonuke I don't know if it does help. It risks turning one simple sentence into ten. That is. 

1) creating an excuse to not cite it doesn't seem kind.
2) however, if we add operation history, then we don't have any need to include the citation.
3) So, if we add operation history, but also want to cite poinssot, so that no one is offended, a need arises to write a whole bunch of sentences about what kinds of metrics come from what kind of raw data.

**Comment by gonuke** [comment]
I guess I missed the context of how we would cite Poinssot.  I was thinking option (3) but without as much connection between which raw data and which metrics.

**Comment by katyhuff** [comment]
@gonuke let me see what I can do on that front.

**Comment by gidden** [comment]
Doing a full cyclus/cycamore recycle simulation will require 1.3, so I
think all relevant changes should be included. However, I think the
reviewer was implying that it was unclear what a BatchReactor was, which I
agree with Katy: it is literally defined in a "term: definition" table.

On Tue, May 12, 2015 at 12:32 PM, Paul Wilson <notifications@github.com>
wrote:

> Regarding BatchReactor: If it doesn't produce substantive changes, I think
> it's fair game to update this section to be consistent with 1.3.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/fundamentals-paper/issues/115#issuecomment-101359685>
> .
>



-- 
Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192

**Comment by katyhuff** [comment]
@gidden , I just went ahead and dropped batchreactor to make things 1.3 compatible. Should I add it back?

**Comment by gidden** [comment]
Nope I think that's fine.

**Comment by katyhuff** [comment]
closed with #131
